.drone.yml

```diff
@@ -14,7 +21,15 @@ pipeline:
     commands:
       docker build -t triton:${DRONE_BRANCH}-${DRONE_BUILD_NUMBER}-${DRONE_COMMIT_SHA:0:8} -f Dockerfile.dev .
 
+  test-gcloud:
+    image: alpine:3.5
+    commands:
+      - ping -c 3 gcloud
```

Comment:
is this just a placeholder for a future integration test against the `gcloud-pubsub-emulator`?

Reply:
No.

DNS in docker is sometimes flaky.  This check helps weed out cases which would result false positives in the integration tests which rely on gcloud mapping to the service container.

Reply:
ah, gotcha.